src/_pytest/logging.py

```diff
@@ -15,7 +15,7 @@
 from _pytest.config import create_terminal_writer
 from _pytest.pathlib import Path
 
-DEFAULT_LOG_FORMAT = "%(filename)-25s %(lineno)4d %(levelname)-8s %(message)s"
+DEFAULT_LOG_FORMAT = "%(levelname)-8s %(name)s:%(filename)s:%(lineno)d %(message)s"
```

Comment:
`(name)` here is the name of the [*logger*](https://docs.python.org/3/library/logging.html#logging.LogRecord), not the module name. It seems it should be `%(module)s` instead: https://docs.python.org/3/library/logging.html#logrecord-attributes

Reply:
The logger name is better IMHO - and what I've meant.. it is typically the module name, but could have more specific info in there.

Reply:
I didn't change anything here in the new version.